api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"partitions", "replicas", "config"})
+public class KafkaMirrorMakerSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private String whitelist;
+
+    private String blacklist;
+
+    private String source;
+
+    private KafkaMirrorMakerTls sourceTls;
+
+    private KafkaMirrorMakerAuthentication sourceAuthentication;
+
+    private String destination;
+
+    private KafkaMirrorMakerTls destinationTls;
+
+    private KafkaMirrorMakerAuthentication destinationAuthentication;
+
+    private Map<String, Object> config;
+
+    private Map<String, Object> additionalProperties;
+
+    @Description("The whitelist of the Mirror Maker")
+    public String getWhitelist() {
```

Comment:
I think you should elaborate on what that whitelist is and what it does. Is it comma separated, pipe separate, a regex? And what does the whitelist do? Either describe it here or link to somewhere it is described.

Reply:
Something like (of course fix for accuracy) : "A comma-separated list of objects that are allowed to do things"